diesel_cli/src/config.rs

```diff
@@ -31,9 +31,14 @@ impl Config {
 
     pub fn read(matches: &ArgMatches) -> Result<Self, Box<Error>> {
         let path = Self::file_path(matches);
-        let mut bytes = Vec::new();
-        fs::File::open(path)?.read_to_end(&mut bytes)?;
-        toml::from_slice(&bytes).map_err(Into::into)
+
+        if path.exists() {
+            let mut bytes = Vec::new();
+            fs::File::open(path)?.read_to_end(&mut bytes)?;
```

Comment:
If you too are randomly wondering if `Read::read_to_end` is cleverly buffered: [It totally is](https://doc.rust-lang.org/1.25.0/src/std/io/mod.rs.html#357)